src/common/i18n/locales/en-US.json

```diff
@@ -1033,7 +1033,7 @@
     "download-our-application": "Download our application",
     "download-our-application-desc": "Looking for a unique experience? The social network with real free speech experience on your mobile phone or desktop. Blockchain's powerful benefits now always under your fingertips.",
     "download-for-windows": "Download for Computer",
-    "download-for-ios":"Download for IOS",
+    "download-for-ios":"Download for Iphone",
```

Comment:
this can stay iOS because app is for iPhone and iPad

Reply:
OK. Changed back to `IOS`